pkg/broker/ingress/multi_topic_decouple_sink.go

```diff
@@ -90,6 +103,36 @@ func (m *multiTopicDecoupleSink) Send(ctx context.Context, broker types.Namespac
 	return err
 }
 
+// EventFilterFunc is used to see if a target is interested in an event.
+// It is used as a vaiable to allow stubbing out in unit tests.
```

Comment:
Can be injected instead?

Reply:
Can u elaborate please?

Reply:
I did consider that, but it seemed a little more complicated than the var. I'm open to it if we feel the function ptr won't work.